dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Converters/GridFormingControl_INIT.mo

```diff
@@ -47,9 +48,9 @@ model GridFormingControl_INIT "Initialization model for the grid forming control
 equation
   /* External loop */
   PRef0Pu = PFilter0Pu;
-  UdFilter0Pu = UFilterRef0Pu;
+  UdFilter0Pu = UFilterRef0Pu - Kff*IdPcc0Pu;
```

Comment:
also we can substract DeltaVVId0 : 

 UdFilter0Pu = UFilterRef0Pu - Kff*IdPcc0Pu **- DeltaVVId0** ;

Could you check if this initialize it better since we have also the expression of DeltaVVId0